### PR TITLE
I'm not quite sure what all of this should mean but these slashes definitely didn't work on linux after simple git clone

### PR DESCRIPTION
**Pull request by Lyapsus**

### CODE REVIEW

**Comment by omar92** [comment]
@wyrde may i ask your support here

File: __init__.py
Comment (omar92) [inline __init__.py:1]: kindly can you do the change on dev , and oull request to dev

Comment (Lyapsus) [reply __init__.py:1]: Greatly sorry, I'm lil unexperienced :) Just opening the folder and chanhing code

**Comment by Lyapsus** [comment]
It seems the SUIT_DIR path is picked up by os.path in every place so these slashes should (?) not be needed even on windows if this is the case

**Comment by omar92** [comment]
because of the src folder i ndeeded to go to the parent dir , will reivew this tmorrow as im sick today

File: __init__.py
Comment (omar92) [reply __init__.py:1]: dont worry , it is my first time in all this too , i use ChatGPT to help me XD , im a C# game developer , i dont know python at all XD

Comment (wyrde) [reply __init__.py:1]: the slashes and dots are in the wrong order.
`SUIT_DIR = SUIT_DIR+"\\.."` means "go to root of filesystem" (the `\`) then go up one more (the `..`)
`SUIT_DIR = SUIT_DIR+"..\"` means "go up one folder/directory"
for completeness:
`SUIT_DIR = SUIT_DIR+".."` = "up one" directory/folder
`SUIT_DIR = SUIT_DIR+"..\.."` = "up two" directory/folder
`SUIT_DIR = SUIT_DIR+".\` = (one dot then a slash) means "same folder"
`SUIT_DIR = SUIT_DIR+".\\.."` = this should be parsed as "go up one from the current folder/directory" but it is kind of iffy.

python *should* change the slash direction (`\` = windows and `/` = linux) automatically with the `os` module. The number of slashes generally doesn't matter as long as they are in the same direction. (`\\\` = `\`) Opposite direction usually means 'escaped` (`\/`) and it doesn't parse the slash as a directory verb but as part of a file name or something.

**Comment by Miyuutsu** [comment]
Was having some issues with this myself. My local fix was to simply comment out `SUIT_DIR = SUIT_DIR+"\\.."`. I am unsure if this is needed in Windows or elsewhere but I've noticed no ill-effect from this.

**Comment by omar92** [comment]
sorry i didnt fix this till now , as i dont have access to Linux machine

**Pull request closed without merging**